src/ucp/rndv/rndv.c

```diff
@@ -1747,6 +1717,9 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_receive, (worker, rreq, rndv_rts_hdr, rkey_buf),
             ucs_assert(rndv_rts_hdr->size <= rreq->recv.length);
             ucp_request_recv_buffer_reg(rreq, ep_config->key.rma_bw_md_map,
                                         rndv_rts_hdr->size);
+
+            rtr_md_map = ep_config->key.rma_bw_md_map &
+                rreq->recv.state.dt.contig.memh->md_map;
```

Comment:
maybe ucp_rndv_req_send_rtr should always use
`(md_map & rreq->recv.state.dt.contig.memh->md_map)`
as the map of sending RTR.
in case of line 1718, we can set `rtr_md_map = ep_config->key.rma_bw_md_map`
and in case of line 1343, we can remove the 2nd `&&` argument